### DISCUSSION TITLE
Error after update docker version 1.28.1 to 1.29.0

### DISCUSSION THREAD

**Discussion by fernandobayit** [dani-garcia/vaultwarden#3849] [Q&A]
Error after update docker version 1.28.1 to 1.29.0

[2023-09-05 00:20:26.368][panic][ERROR] thread 'main' panicked at 'Error running migrations: QueryError(DieselMigrationName { name: "2023-02-18-125735_push_uuid_table", version: MigrationVersion("20230218125735") }, DatabaseError(Unknown, "must be owner of table devices"))': src/db/mod.rs:497

   0: std::backtrace::Backtrace::create

   1: vaultwarden::init_logging::{{closure}}

   2: std::panicking::rust_panic_with_hook

   3: std::panicking::begin_panic_handler::{{closure}}

   4: std::sys_common::backtrace::__rust_end_short_backtrace

   5: rust_begin_unwind

   6: core::panicking::panic_fmt

   7: core::result::unwrap_failed

   8: vaultwarden::db::postgresql_migrations::run_migrations

   9: vaultwarden::db::DbPool::from_config

  10: vaultwarden::main::{{closure}}

  11: tokio::runtime::park::CachedParkThread::block_on

  12: tokio::runtime::context::runtime::enter_runtime

  13: tokio::runtime::runtime::Runtime::block_on

  14: rocket::async_main

  15: vaultwarden::main

  16: std::sys_common::backtrace::__rust_begin_short_backtrace

  17: std::rt::lang_start::{{closure}}

  18: std::rt::lang_start_internal

  19: main

  20: __libc_start_main

  21: <unknown>

* Install method:
Docker

* Clients used:
webvault, Android, iOS

* Reverse proxy and version: 
Caddy

* MySQL/MariaDB or PostgreSQL version: 
PostgreSQL

* Other relevant details:

**Comment by BlackDex** [comment]
Seems your database has some issues with ownership.
I suggest to check your database for issues.

I suggest to search for the error message reported by PostgreSQL to look for a solution.

**Reply by fernandobayit** [reply]
I'm really sorry. You are correct, follow the commands to fix permissions on the database (PostgreSQL)

GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO vaultwarden;
GRANT ALL PRIVILEGES ON ALL SEQUENCES IN SCHEMA public TO vaultwarden;
GRANT ALL PRIVILEGES ON ALL FUNCTIONS IN SCHEMA public TO vaultwarden;

Thanks, regards
